src/main.rs

```diff
@@ -528,6 +528,15 @@ mod backtrace {
     }
 }
 
+fn play_scream() -> Option<()> {
```

Comment:
Since you're using `None` to say there was an error, I think this should instead be `Result<(), ()>`.